students/JaeKim/session02/series.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+"""
+a template for the series assignment
+"""
+
+
+def fibonacci(n):
+    """Returns the nth value of the Fibonacci numbers series
+    """
+    if n == 0:
+        return 0
+    if n == 1 or n == 2:
+        return 1
+    return fibonacci(n - 1) + fibonacci(n - 2)
+
+def lucas(n):
+    """Returns the nth value of the Lucas number series
+    """
+    if n == 0:
+        return 2
+    if n == 1:
+        return 1
+    return lucas(n - 1) + lucas(n - 2)
+
+def sum_series(n, n0=0, n1=1):
```

Comment:
you've missed the point here -- you want sum_series to be able to compute a series with any two numbers to start it off -- fib and lucas are only two possibilities...

look at how similar the two functions are -- how would you generalize that to start off with any two numbers?